src/lib.rs

```diff
@@ -161,7 +163,7 @@ where
     let sz = mem::size_of::<T>();
     let max_bytes = if max_bytes < sz { sz } else { max_bytes };
     let fs_lock = sync::Arc::new(sync::Mutex::new(private::FsSync::new(cap)));
-    let sender = try!(Sender::new(name, &snd_root, max_bytes, fs_lock.clone()));
+    let sender = try!(Sender::new(name, &snd_root, max_bytes, Arc::clone(&fs_lock)));
```

Comment:
Not sure, why you have done this.
According to https://doc.rust-lang.org/std/sync/struct.Arc.html
these two syntaxes are equivalent

Reply:
Clippy recommendation. https://github.com/rust-lang-nursery/rust-clippy/issues/1645